stix_shifter_modules/qradar/configuration/qradar_config.json

```diff
@@ -40,6 +41,13 @@
         "selfSignedCert": {
             "type": "password",
             "optional": true
+        },
+        "proxy": {
+            "type": "text",
```

Comment:
proxy is a section that has to include other fields. https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter_modules/qradar/stix_transmission/arielapiclient.py#L32-L33

Reply:
yes. will update